tests/test_pipelines.py

```diff
@@ -207,6 +207,15 @@ def test_load_custom_pipeline(self):
         # under https://huggingface.co/hf-internal-testing/diffusers-dummy-pipeline/blob/main/pipeline.py#L24
         assert pipeline.__class__.__name__ == "CustomPipeline"
 
+    def test_load_custom_github(self):
+        pipeline = DiffusionPipeline.from_pretrained(
+            "google/ddpm-cifar10-32", custom_pipeline="one_step_unet", custom_revision="0.10.2"
```

Comment:
How does this test verify that the pipeline came from `0.10.2`? We could, for example:
- Download from a version where the pipeline did not exist (should fail).
- Download from the first version where it does (succeeds).

Reply:
Very good point! Will adapt the test :-)